solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -257,6 +271,12 @@ describe("RandomBeacon - Group Creation", () => {
 
           context("when dkg result was challenged", async () => {
             beforeEach(async () => {
+              // Make the challenge justified by forcing the pool to return
+              // an other actual group than the submitted one.
```

Comment:
s/an other actual group/another group ?

I think this will be easier to reason about

Reply:
See https://github.com/keep-network/keep-core/pull/2695/commits/0d6f0ab25225ac5f2407f3a0d07b2b922fd33c89